### PR TITLE
Web3 1034 zkverify integration

### PR DESCRIPTION
**Pull request by rushby**
### Description

- Full integration using latest zkverifyjs package stored as tgz file
- Updated Readme with cloudflare information
- Fixed Typescript linting errors
- Addressed comments
- Configured cloudflare deployment

### CODE REVIEW

**Comment by rushby** [comment]
Also worth noting no effort was made to fix all the typescript and IDE flagged issues with the original code.

File: packages/app/src/Constants.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
-export const APP_NAME = 'zkSNARK Sudoku';
+export const APP_NAME = 'zkSNARK Sudoku x zkVerify';
+export const ORIGINAL_APP = 'zkSNARK Sudoku'
 export const APP_DESCRIPTION =
-  'This is zkSNARK Sudoku play and verify application.';
+  'This is zkSNARK Sudoku: play and verify the proof on zkVerify.';
 
 export const GMAIL = 'mailto:dany.armstrong90@gmail.com';
 export const LINKEDIN = 'http://www.linkedin.com/in/daniel-armstrong90';
```
Comment (yurikoinaba) [inline packages/app/src/Constants.tsx:7]: `export const GITHUB_PROJECT = 'https://github.com/HorizenLabs/zksnark-sudoku';`

This is used on the GitHub logo on the header

File: packages/app/src/layout/AppLayout.tsx [hunk 2]
```diff
@@ -1,76 +1,101 @@
...
+    ORIGINAL_APP,
 } from '../Constants';
 import logo from '../images/sudoku.png';
+import ConnectWalletButton from '../components/ConnectWalletButton';
 
 export default function AppLayout({ children }: { children: ReactNode }) {
-  return (
-    <Row>
-      <Col span={24}>
+    const [selectedAccount, setSelectedAccount] = useState<string | null>(null);
```
Comment (yurikoinaba) [inline packages/app/src/layout/AppLayout.tsx]: use context

File: DEV_README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Install
+
+1. The legacy peer deps is required on install:
+
+```shell
+npm install --legacy-peer-deps
+```
+
+2. Development using a locally built zkverifyjs tgz file (placed in `packages/zkverifyjs`) might require removing and reinstalling dependencies
```
Comment (yurikoinaba) [inline DEV_README.md]: Let's use the npm package once it's published

File: packages/app/src/Constants.tsx (hunk 1 above)
Comment (rushby) [reply packages/app/src/Constants.tsx:7]: I left that on purpose along with the original devs details wasn't sure how much I should change or keep the reference to original project.

**Review by yurikoinaba — APPROVED** [review]

**Pull request merged**